nodes/src/nodes/audio_tts/IGlobal.py

```
@@ -0,0 +1,273 @@
+                    os.remove(full_path)
+            except OSError:
+                # Best-effort cleanup only.
+                continue
+
+    def validateConfig(self):
+        raw, cfg = self._resolve_merged_config()
+        engine = self._engine_from_merged_cfg(cfg)
+        api_key = self._resolve_cloud_api_key(cfg, raw, engine)
+        if engine in ('elevenlabs', 'openai') and not api_key:
+            env_hint = 'OPENAI_API_KEY' if engine == 'openai' else 'ELEVENLABS_API_KEY'
+            raise Exception(f'Engine "{engine}" requires api_key in node config or {env_hint} in the engine environment')
```

Ruff (0.15.7)

[warning] Create your own exception

(TRY002)